test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -1027,13 +1027,15 @@ TEST_F(AtenXlaTensorTest, TestSort) {
   for (int k = 1; k <= 3; ++k) {
     for (int dim = 0; dim < 3; ++dim) {
       for (bool descending : {false, true}) {
-        auto b = torch::sort(a, dim, descending);
-        ForEachDevice([&](const torch::Device& device) {
-          torch::Tensor xla_a = CopyToDevice(a, device);
-          auto xla_b = torch::sort(xla_a, dim, descending);
-          AllClose(std::get<0>(b), std::get<0>(xla_b));
-          AllEqual(std::get<1>(b), std::get<1>(xla_b));
-        });
+        for (bool stable : {false, true}) {
+          auto b = torch::sort(a, dim, descending);
```

Comment:
nit, let's use `torch::Tensor` directly here since you also touch this part of the code 😄

Reply:
Thanks for the comment, could you clarify little bit for me?

Reply:
oh I meant instead of 
```
auto b = torch::sort(a, dim, descending);
```

we should just do 
```
torch::Tensor b = torch::sort(a, dim, descending);

```

Reply:
Actually, ::sort returns a tuple `'::std::tuple<at::Tensor, at::Tensor>'`, guess that's why `auto` was used originally? lol

Reply:
oh, nvm then lol